resources/views/components/checkin.blade.php

```diff
@@ -42,7 +42,7 @@
                                 <input type="text" class="form-control" name="checkin_qty" aria-label="checkin_qty" value="{{ old('assigned_qty', $component_assets->assigned_qty) }}">
                             </div>
                             <div class="col-md-9 col-md-offset-2">
-                            <p class="help-block">{{ trans(admin/components/general.checkin_limit) }}: {{ $component_assets->assigned_qty }}</p>
+                            <p class="help-block">{{ trans(admin/components/general.checkin_limit, ['assigned_qty' => '$component_assets->assigned_qty']) }}</p>
```

Comment:
Looks like we still need enclosing apostrophes here tho:

`{{ trans(admin/components/general.checkin_limit, ['assigned_qty' => '$component_assets->assigned_qty']) }}

should be 

`{{ trans('admin/components/general.checkin_limit', ['assigned_qty' => '$component_assets->assigned_qty']) }}`

The key we present in the `trans()` bit should always be enclosed in apostrophes.

Reply:
And actually we wouldn't need them enclosed with the variable part. 

`{{ trans('admin/components/general.checkin_limit', ['assigned_qty' => $component_assets->assigned_qty]) }}`

When we're passing a variable, enclosing it on quotes will cause it to return the literal string of `$component_assets->assigned_qty`, instead of the *value* of `$component_assets->assigned_qty`.